automation_tools/__init__.py

```diff
@@ -1809,8 +1825,18 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
         The url for capsule repo from latest satellite compose.
     TOOLS_URL
         The url for sat-tools repo from latest satellite compose.
+    FROM_VERSION:
+        The satellite current version. To differ representation of
```

Comment:
Replace with the following?
`Current Satellite version - to differentiate default organization. e.g., '6.1', '6.0'`